request/src/main/java/com/vimeo/networking2/VimeoService.kt

```diff
@@ -0,0 +1,478 @@
+/*
+ * Copyright (c) 2020 Vimeo (https://vimeo.com)
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package com.vimeo.networking2
+
+import com.vimeo.networking2.internal.VimeoCall
+import com.vimeo.networking2.params.BatchPublishToSocialMedia
+import com.vimeo.networking2.params.ModifyVideoInAlbumsSpecs
+import com.vimeo.networking2.params.ModifyVideosInAlbumSpecs
+import okhttp3.CacheControl
+import retrofit2.http.Body
+import retrofit2.http.DELETE
+import retrofit2.http.GET
+import retrofit2.http.Header
+import retrofit2.http.Headers
+import retrofit2.http.PATCH
+import retrofit2.http.POST
+import retrofit2.http.PUT
+import retrofit2.http.Path
+import retrofit2.http.QueryMap
+import retrofit2.http.Url
+
+/**
+ * Retrofit service for the Vimeo API.
+ */
+@Suppress("unused", "UndocumentedPublicFunction", "ComplexInterface")
+interface VimeoService {
+
+    @GET("me/connected_apps")
+    fun getConnectedAppList(
+        @Header(AUTHORIZATION) authorization: String,
+        @Header(CACHE_CONTROL) cacheControl: CacheControl?
+    ): VimeoCall<ConnectedAppList>
+
+    @GET("me/connected_apps/{type}")
+    fun getConnectedApp(
+        @Header(AUTHORIZATION) authorization: String,
+        @Path("type") type: String,
+        @Header(CACHE_CONTROL) cacheControl: CacheControl?
+    ): VimeoCall<ConnectedApp>
+
+    @PUT("me/connected_apps/{type}")
+    @Headers("Cache-Control: no-cache, no-store")
```

Comment:
`"Cache-Control: no-cache, no-store"` gets repeated a couple of times and could be a `const` instead